### DISCUSSION TITLE
Chia stolen

### DISCUSSION THREAD

**Discussion by SidRajvosa** [Chia-Network/chia-blockchain#15332] [Support]
Hi,
My friend wrote here on my behalf, but since we made no progress, I decided to register here to get help. Long story short, we have been robbed as all of our chia has been transferred to someone's wallet without our knowledge or consent. 
Below is his post that explains everything that happened. Can we get support before we seek legal advice?

Dear all,

Please try to help me, my friend have issues with chia coins, he used space pool for mining and till today he mined around 32 chia coin, please check his deposit addresses: 

xch1vajhn5menmrqnw2cpstmj8t8g80ywrl2lj0dwwjae9325cfgvwhsl2dxv0

And 

xch19uef872gg68nrfre9905ce9ssjr4xt700x4au92ysp2k42dl4rwq2c9xxz

But after he checked his wallets he founds that coins are sent to this address:

xch1w48cgmqt4qjpd2ktc3xxk8awn5399usk8l0ehdasqjamgw0v55ysawak45

As we checked comments, we found that also some people had the same issue.

Nobody has his account info and how is this possible to do it?

Please check and send me your comments.

Please help.

Regards,

**Comment by SidRajvosa** [comment]
Anyone?

**Comment by shaneo257** [comment]
sorry theres not much else anyone can do on a permissionless blockchain, if they have definately gone, then they're gone
